.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+  "editor.codeActionsOnSave": {
+    "source.organizeImports.biome": "explicit"
```

Comment:
Just to be sure, this is covered by CI and the format command? So I can use neovim 😆

Reply:
It is yes don't worry :) Feel free to use whatever IDE you want, this file simply won't do anything if you're not on VSCode but it's not mandatory.